src/services/campaign-delivery.js

```diff
@@ -111,15 +111,28 @@ module.exports = {
     return this.selectCampaigns(campaigns, limit);
   },
 
+  /**
+   * Returns the impression reserve based on the supplied placement and account settings
+   *
+   * @param placement The placement model
+   * @param account The account model
+   */
+  calculateImpressionReserve({
+    placement,
+    account,
+  }) {
+    const rp = parseInt(placement.get('reservePct'), 10);
+    const ap = account.get('settings.reservePct');
+    return (Number.isInteger(rp) ? rp : (ap || 0)) / 100;
```

Comment:
A float would also be acceptable

Reply:
The GraphQL definitions for the [account](https://github.com/parameter1/fortnight-graph/blob/master/src/graph/definitions/account.graphql#L24) and [placement](https://github.com/parameter1/fortnight-graph/blob/master/src/graph/definitions/placement.graphql#L26) are both `Int` types, so I don't think floats _should_ be accepted. I could swap this to use `Number.isFinite` if we want to allow it, though.